packages/karma-ui5-transpile/lib/index.js

```diff
@@ -75,21 +74,23 @@ function createPreprocessor(config, logger) {
 				return plugin.file.request === "istanbul";
 			})
 		) {
+			const istanbulConfig = {
+				include: ["**/*"],
+				exclude: []
+			};
+			// apply the `instrumenterOptions` for istanbul from `coverageReporter` in the `karma.config`
+			const instrumenterOptionsIstanbul = config.coverageReporter?.instrumenterOptions?.istanbul;
+			if (instrumenterOptionsIstanbul && typeof instrumenterOptionsIstanbul === "object") {
+				Object.assign(istanbulConfig, config.coverageReporter.instrumenterOptions.istanbul);
```

Comment:
Here you could use `instrumenterOptionsIstanbul` instead of `config.coverageReporter.instrumenterOptions.istanbul`

Reply:
That was actually the reason for creating `instrumenterOptionsIstanbul` as a variable ... seems like I have been too tired yesterday